apps/deploy-web/src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -158,17 +153,25 @@ export const WalletProvider = ({ children }) => {
   }
 
   async function loadWallet(): Promise<void> {
-    let currentWallets = getStorageWallets();
+    const networkId =
```

Comment:
if we switch from a net that managed wallets are not supposed to operate on the below updates are run on the prev net and then reload happens. So now it would be running on the net that is to be changed to.